.github/workflows/test.yml

```diff
@@ -76,11 +76,34 @@ jobs:
           npx playwright install
           npx playwright install-deps
 
-      - run: node . --suiteName basic-js --runCfgPath ./tests/fixtures/basic-js/sauce-runner.json
+      - run: node . --suiteName basic-js --runCfgPath ./tests/fixtures/local/basic-js/sauce-runner.json
         env:
           SAUCE_VM: true
         name: Run test locally
 
+  cucumber-build-and-test-local:
+    runs-on: windows-latest
+    needs: [test]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
```

Comment:
It has been added, but looks it is mainly without any effects. It's now present on each task of the workflow.

Reply:
Perhaps what takes the longest to install here are the playwright browsers, which would not be cached, since they are outside of npm.
We could try and cache the playwright "browser cache" folder via https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows